tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor/EventProcessing/IssueProcessing.cs

```diff
@@ -178,7 +178,7 @@ public static async Task InitialIssueTriage(GitHubEventClient gitHubEventClient,
                                     }
                                 }
                                 // If the issue had a valid assignee and only labels provided add the comment
-                                if (hasValidAssignee && triageOutput.Suggestion == null && triageOutput.Solution == null)
+                                if (hasValidAssignee && triageOutput.AnswerType == "none" && triageOutput.Labels.Count > 0)
```

Comment:
I'd add a `string.IsNullOrEmpty` guard for "AnswerType" here too - just to be safe.